MapboxNavigation/DayStyle.swift

```diff
@@ -3,12 +3,13 @@ import Mapbox
 
 
 extension UIColor {
-    class var defaultRouteCasing: UIColor { get { return .defaultTintStroke } }
+    class var defaultRouteLine: UIColor { ColorManager.shared.palette.routeLineColor }
```

Comment:
Dit zit in DayStyle.swift, is er ook iets van een NightStyle? Waar dit misschien ook in zou moeten?

Reply:
De NightStyle klasse zit in de DayStyle klasse hoe verwarrend het ook is